### PR TITLE
Feature/improved create pbc task

### PR DESCRIPTION
**Pull request by newLoki**
Creating a PBC through the sdk (`spryker-sdk generate:php:pbc`) only created an outdated PBC boilerplate that wasn't even compatible with the newest docker/sdk and the spryker/aop-sdk.
Goal here is to have a fully working project set up after the task has run.

This pull request will:

- fix naming of the composer project
- update all depenencies, based on PHP 7.4 (optional `spryker-sdk generate:php:pbc --pbc_php_version=8.0` will use PHP 8)
- Ensure the docker/sdk environment of the PBC has a different namespace as the spryker demoshop as they would share the same mutagen/docker-sync container if the namespace is the same and code would be synchronized between different projects
- Install docker/sdk to the newly created PBC

Another thing I noticed using the SDK, the build https://github.com/spryker-sdk/sdk/releases/tag/0.1.1 is not installable at all. The current master is installable, so it would be great to build a new installer and publish it.

### CODE REVIEW

File: src/Extension/Tasks/Commands/NextPbcStepsCommand.php [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+class NextPbcStepsCommand implements ExecutableCommandInterface
+{
+    /**
+     * @param \SprykerSdk\SdkContracts\Entity\ContextInterface $context
+     *
+     * @return \SprykerSdk\SdkContracts\Entity\ContextInterface
+     */
+    public function execute(ContextInterface $context): ContextInterface
+    {
+        $message = 'Run "cd %s && docker/sdk boot -s deploy.dev.yml" and follow the instructions to spin up the local environment and follow the instructions';
```
Comment (stereomon) [inline src/Extension/Tasks/Commands/NextPbcStepsCommand.php:25]:
```suggestion
        $message = 'Run "cd %s && docker/sdk boot -s deploy.dev.yml" and follow the instructions to spin up the local environment';
```

File: src/Extension/Tasks/Commands/NextPbcStepsCommand.php [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+namespace SprykerSdk\Sdk\Extension\Tasks\Commands;
+
+use SprykerSdk\Sdk\Core\Domain\Entity\Message;
+use SprykerSdk\SdkContracts\Entity\ContextInterface;
+use SprykerSdk\SdkContracts\Entity\ConverterInterface;
+use SprykerSdk\SdkContracts\Entity\ExecutableCommandInterface;
+use SprykerSdk\SdkContracts\Entity\MessageInterface;
+
+class NextPbcStepsCommand implements ExecutableCommandInterface
```
Comment (stereomon) [inline src/Extension/Tasks/Commands/NextPbcStepsCommand.php:16]: A class doc block could help to understand what this class is for.

**Review by stereomon — CHANGES_REQUESTED** [review]

**Comment by stereomon** [comment]
@vol4onok Can you take care of getting this merged as well?

**Comment by vol4onok** [comment]
Thanks for the PR. it was processed here with merging and fixing https://github.com/spryker-sdk/sdk/pull/103

**Pull request merged**